client/blocks/login/style.scss

```diff
@@ -9,6 +9,60 @@
 	}
 }
 
+.layout.is-jetpack-woocommerce-flow {
+	.login__jetpack-logo {
+		text-align: center;
+		border-bottom: 1px solid #1f112e;
```

Comment:
Could this hardcoded hex value be switched to CSS variable?

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/33073/commits/ec4b2b0da4a0c1137a0b4760fb87999cff3137aa.